capstone/src/components/User/Experience/Performance/Performance.jsx

```diff
@@ -0,0 +1,61 @@
+import { Box, HStack, Heading, Text } from '@chakra-ui/react';
+import Graph from './Graph';
+import Comment from './Comment';
+
+function determineColor(value) {
+  switch (value) {
+    case 1:
+      return '#E53E3E';
+    case 2:
+      return '#DD6B20';
+    case 3:
+      return '#ECC94B';
+    case 4:
+      return '#68D391';
+    case 5:
+      return '#38B2AC';
+    default:
+      return '#0000000';
+  }
+}
+
+export default function Performance() {
```

Comment:
I guess these are copied from the library examples, but it'd be good to use names specific to this app.